src/OrchardCore.Modules/OrchardCore.Users/ViewModels/ChangeEmailViewModel.cs

```diff
@@ -1,13 +1,21 @@
+using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Localization;
-using MimeKit;
+using Microsoft.Extensions.Localization;    
+using OrchardCore.Email;
 
 namespace OrchardCore.Users.ViewModels
 {
     public class ChangeEmailViewModel : IValidatableObject
```

Comment:
Do we really need `IValidatableObject`? `IEmailAddressValidator` can handle the validation logic, so you can check directly in the `POST`